interface/backend/api/serializers.py

```diff
@@ -54,3 +54,11 @@ class Meta:
         read_only_fields = ('created',)
 
     centroid = ImageLocationSerializer()
+
+    def create(self, validated_data):
```

Comment:
This seems... quite long-winded. Why not just:

```
def create(self, validated_data):
    return Nodule.objects.create(
        case=validated_data['case'],
        candidate=validated_data['candidate'],
        centroid=ImageLocation.objects.create(**validated_data['centroid']),
    )

It's currently quite hard to follow with all the variable assignments to sliiiiiightly different names if you know what I mean?

Reply:
Nice. Short and exactly what I tried to achieve ... Thanks!